### ISSUE TITLE
additional system packages installed on ubuntu 14.04 (Trac #1511)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1503]
I setup a new home machine, running Ubuntu14.04, and followed
these instructions:
http://software.icecube.wisc.edu/offline_trunk/projects/cmake/platforms.html#ubuntu-or-debian-variants

Checking after cmake what was mising, I found adding a few more packages was necessary for icerec and offline:

apt-get install libstarlink-pal-dev
apt-get install python-sphinx
apt-get install libopenblas-dev

Cheers
Erik

Migrated from https://code.icecube.wisc.edu/ticket/1511
```json
{
    "status": "closed", 
    "changetime": "2019-01-12T00:06:53", 
    "description": "I setup a new home machine, running Ubuntu14.04, and followed\nthese instructions:\nhttp://software.icecube.wisc.edu/offline_trunk/projects/cmake/platforms.html#ubuntu-or-debian-variants\n\nChecking after cmake what was mising, I found adding a few more packages was necessary for icerec and offline:\n\napt-get install libstarlink-pal-dev\napt-get install python-sphinx\napt-get install libopenblas-dev\n\nCheers\nErik", 
    "reporter": "blaufuss", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1547251613874407", 
    "component": "cmake", 
    "summary": "additional system packages installed on ubuntu 14.04", 
    "priority": "major", 
    "keywords": "", 
    "time": "2016-01-18T16:07:54", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160118T16:11:31: 
* david.schultz changed status from "new" to "accepted"
* david.schultz changed milestone from "" to "January 2016"
* david.schultz changed priority from "normal" to "major"
* david.schultz changed owner from "" to "david.schultz"

**Comment by icecube-trac** [comment]
Trac update at 20160118T16:25:33: 
* david.schultz commented:

> In changeset 2385:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2385"
> more packages for ubuntu 14.04. closes #1503
> ```


* david.schultz changed resolution from "" to "fixed"
* david.schultz changed status from "accepted" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20160318T21:14:09: 
* olivas commented:

> Milestone January 2016 deleted


* olivas changed milestone from "January 2016" to ""

**Comment by icecube-trac** [comment]
Trac update at 20190112T00:06:53: nwhitehorn commented:

> In changeset 170167:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="170167"
> more packages for ubuntu 14.04. closes #1503
> ```
